### PR TITLE
Support Multi Flow

### PR DESCRIPTION
**Pull request by EugeneButusov**

### CODE REVIEW

File: test/apiService.unit.ts [hunk 1]
```diff
@@ -96,7 +96,7 @@ describe('apiService', () => {
     };
 
     describe('when right signature', () => {
-      const context: Context = withApiService((createContext()));
```
Comment (EugeneButusov) [inline test/apiService.unit.ts:99]: extra parentheses, not sure why was introduced.

File: test/__snapshots__/apiServiceV1.unit.ts.snap [hunk 2]
```diff
@@ -41,27 +41,3 @@ Array [
   ],
 ]
 `;
-
-exports[`apiServiceV1 #fetchResource with right auth should call callHttp 1`] = `
```
Comment (EugeneButusov) [inline test/__snapshots__/apiServiceV1.unit.ts.snap:45]: jest marked it as obsolete test.

File: src/models/v2/SendInputResponse.ts [hunk 3]
```diff
@@ -36,9 +36,14 @@ interface InputError {
...
-  result?: boolean;
+interface ErrorInputResult {
   error: InputError;
 }
 
+interface SuccessInputResult {
+  result: boolean;
+}
+
+export type InputResult = ErrorInputResult | SuccessInputResult;
```
Comment (EugeneButusov) [inline src/models/v2/SendInputResponse.ts:47]: to support `either/or` logic for result and error fields exclusive displaying.

File: test/__snapshots__/apiServiceV1.unit.ts.snap (hunk 2 above)
Comment (hostmepanda) [reply test/__snapshots__/apiServiceV1.unit.ts.snap:45]: aha, due to rename ))

File: src/apiService.ts [hunk 4]
```diff
@@ -96,11 +101,15 @@ class ApiService {
    * @throws ErrorResponse if we get http error
    */
   public async createIdentity(metadata?: IdentityMetadata): Promise<IdentityResource> {
+    const body: Record<string, any> = { metadata };
+    if (this.flowId) {
```
Comment (vedi) [inline src/apiService.ts:105]: cannot `flowId` be `0`? I think it's better to check for `undefined`

File: src/apiService.ts [hunk 5]
```diff
@@ -96,11 +101,15 @@ class ApiService {
    * @throws ErrorResponse if we get http error
    */
   public async createIdentity(metadata?: IdentityMetadata): Promise<IdentityResource> {
+    const body: Record<string, any> = { metadata };
```
Comment (vedi) [inline src/apiService.ts:104]: I think it's better to define a strict type here instead of `Record`.

File: src/apiService.ts (hunk 4 above)
Comment (EugeneButusov) [reply src/apiService.ts:105]: actually there is hexadecimal string, not integer.

File: src/apiService.ts (hunk 5 above)
Comment (EugeneButusov) [reply src/apiService.ts]: @vedi, fixed now :)

**Pull request merged**